src/moi.jl

```diff
@@ -815,21 +815,24 @@ function pass_names(dest, src, map)
     end
 end
 
-function append_to(dest::MOI.ModelLike, src::MOI.ModelLike, idxmap, copy_names::Bool; allow_single_bounds::Bool = true)
+function append_to(dest::MOI.ModelLike, src::MOI.ModelLike, idxmap, copy_names::Bool, 
+    filter_constraints::Union{Nothing, Function}=nothing; allow_single_bounds::Bool = true)
     # MOI.empty!(dest)
 
     # idxmap = MOIU.IndexMap()
 
     vis_src = MOI.get(src, MOI.ListOfVariableIndices())
-    constraint_types = MOI.get(src, MOI.ListOfConstraints())
-    single_variable_types = [S for (F, S) in constraint_types
-                             if F == MOI.SingleVariable && allow_single_bounds]
-    vector_of_variables_types = [S for (F, S) in constraint_types
-                                 if F == MOI.VectorOfVariables]
-
+    # index_map_for_variable_indices only initializes the data structure
+    # idxmap = index_map_for_variable_indices(vis_src)
```

Comment:
Remove?

Reply:
Grazie,
I will add an extra comment explaining why to keep these comments.
This is basically a copy os `MOIU.default_copy_to`, to the comments hight light what is equal and what is different.